src/schema/Transfer/dataloaders/QuoteEvents.ts

```diff
@@ -0,0 +1,54 @@
+/**************************************************************************
+ *  (C) Copyright Mojaloop Foundation 2020                                *
+ *                                                                        *
+ *  This file is made available under the terms of the license agreement  *
+ *  specified in the corresponding source code repository.                *
+ *                                                                        *
+ *  ORIGINAL AUTHOR:                                                      *
+ *       Yevhen Kyriukha <yevhen.kyriukha@modusbox.com>                   *
+ **************************************************************************/
+
+import { Context } from '@app/context';
+import DataLoader from 'dataloader';
+
+const ID = Symbol();
+
+// Function to fetch all quote events for a transactionId
+const findQuoteEvent = async (ctx: Context, transactionId: string) => {
+  const events = await ctx.eventStore.reportingData.findMany({
+    where: {
+      metadata: {
+        equals: {
+          reporting: {
+            eventType: 'Quote',
+            transactionId: transactionId,
```

Comment:
We need to check whether we have index on these fields

Reply:
Prism doesn't support indexing for nested fields so i am not able to add indexing for these.